package.json

```diff
@@ -7,7 +7,7 @@
     "doc": "doc"
   },
   "scripts": {
-    "test": "npm run eslint && npm run mocha",
+    "test": "npm run eslint && npm run mocha && gulp build",
```

Comment:
Does any step of the deployment process run this?

Reply:
This will run before each _continuous_ deployment by Travis CI (also see below). The CI run will not continue with deployment if the `npm test` run fails. Running only the deploy script yourself will not run this.